README.md

```diff
@@ -69,15 +66,44 @@ qemu-system-aarch64 \
     -snapshot output/qcow2/disk.qcow2
 ```
 
-## Volumes
-- `/output` - used for output files
-- `/store` - used for the osbuild store
-- `/rpmmd` - used for the dnf-json rpm metadata cache
+## 📝 Arguments
+
+`bootc-image-builder` always requires one positional argument: The container image reference.
+
+The following tables show all optional arguments:
+
+| Argument     | Description                                                      | Default Value |
+|--------------|------------------------------------------------------------------|:-------------:|
+| **--config** | Path to a [build config](#-build-config)                         |       ❌       |
+| --tls-verify | Require HTTPS and verify certificates when contacting registries |    `true`     |
+| **--type**   | [Image type](#-image-types) to build                             |    `qcow2`    |
+
+## 💾 Image types
+
+The following image types are currently available via the `--type` argument:
 
-## Adding a user
-`bootc-image-builder` accepts a `--config` option. `--config` needs to be a path to a JSON formatted file.
+| Image type            | Target environment                                                                    |
+|-----------------------|---------------------------------------------------------------------------------------|
+| `ami`                 | [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
+| `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
 
-Example of such a config:
+## 💽 Volumes
+
+The following volumes can be mounted inside the container:
+
+| Volume    | Purpose                                                | Required |
+|-----------|--------------------------------------------------------|:--------:|
+| `/output` | Used for storing the resulting artifacts               |    ✅     |
+| `/store`  | Used for the [osbuild store](https://www.osbuild.org/) |    ❌     |
```

Comment:
It's probably silly, but I don't like the red X. It draws my attention, and makes me have to think twice about what it means. I think either leaving it blank, or a simple 'No' would be better.

Reply:
It isn't silly at all! I actually had the same feeling even before you stumbled upon it. I left only the green checkmarks, and replaced the red crosses with a plain "No", exactly as you suggested. :) Thanks!